### ISSUE TITLE
Could not find or load main class ISOViewer

### ISSUE THREAD

**Issue by SulfredLee** [sannies/isoviewer#16]
I am using ubuntu 18.04
```
java -version
openjdk version "1.8.0_212"
OpenJDK Runtime Environment (build 1.8.0_212-8u212-b03-0ubuntu1.18.04.1-b03)
OpenJDK 64-Bit Server VM (build 25.212-b03, mixed mode)
```
When I run
`java -jar isoviewer-2.0.2-jfx.jar`
I get
`Error: Could not find or load main class ISOViewer`

I see other issue talking about the version of the java make this problem. But I am using java 8 already.
[https://github.com/sannies/isoviewer/issues/5](url)
[https://github.com/sannies/isoviewer/issues/15](url)

I am not very familiar with the version control of the java, so how can I run isoviewer?

Thank you very much.

**Comment by jurisbu** [comment]
Yeah, this is a bit annoying that there are no instruction how to actually run this fine application! Would be great to have some "grandma" instructions.
